altair/utils/save.py

```diff
@@ -45,7 +88,7 @@ def save(
         the format to write: one of ['json', 'html', 'png', 'svg', 'pdf'].
```

Comment:
```suggestion
        the format to write: one of ['json', 'html', 'png', 'svg', 'pdf', 'vega', 'vega-lite'].
```

Only add the `vega-lite` suggestion if it is also supported in the rest of `save`